server/bin/pbench-backup-tarballs.py

```diff
@@ -46,15 +46,15 @@ def __init__(self, ntotal=0, nbackup_success=0, nbackup_fail=0,
 def sanity_check(lb_obj, s3_obj, config, logger):
     # make sure archive is present
     archive = config.ARCHIVE
+    archivepath= os.path.realpath(archive)
 
-    if not os.path.realpath(archive):
-        logger.error(
-            'The ARCHIVE directory {}, does not resolve to a real location'.format(archive))
+    if not archivepath:
+        logger.error('The ARCHIVE directory {}, does not resolve to a real location', archive)
```

Comment:
Just nitpicking here but for the sake of readability I would rather use the python3 f-strings.
logger.error(f'The ARCHIVE directory {archive}, does not resolve to a real location')

Reply:
If the logging level is set such that the message won't be emitted, won't using f-strings incurs all the formatting without being useful?

As a rule, we are trying to avoid pre-formatted strings in log messages.